src/main/webapp/clusters-view/clusters-view.html

```diff
@@ -1,46 +1,72 @@
+<div class="container-fluid">
+  <div class="row header-row">
+    <div class="col-sm-8 col-md-9">
+      <ol class="breadcrumb">
+        <li>Cache containers</li>
+      </ol>
+      <h1>Cache containers</h1>
+      <br>
+      <table class="table ispn-aligned">
+        <tr>
+          <th>Status</th>
+          <th>Name</th>
+          <th># Caches</th>
+          <th>Clustering info <span class="icon-class pficon-info"></span></th>
+          <th>Endpoints <span class="icon-class pficon-info"></span></th>
+          <th>Sites <span class="icon-class pficon-info"></span></th>
+        </tr>
 
-    <!-- TOP ROW #1 START -->
-    <div class="row">
-        <div class="panel panel-default">
-            <div class="panel-heading">Clusters</div>
-            <div class="list-group">
-                <div class="list-group-item row" ng-repeat="cluster in clusters">
-                    <div class="col-md-4">
-                        <a href="#/cluster/{{cluster.name}}"><h4>{{cluster.name}}</h4>
-                            <div>
-                                <p class="fa fa-check-square fa-1x"
-                                   ng-class="(cluster.isAvailable())?'available':'unavailable'">
-                                    {{cluster.isAvailable()?cluster.availability:'UNAVAILABLE'}}</p>
-                            </div>
-                        </a>
-                    </div>
-                    <div class="col-md-4">{{cluster.getNodes().length}} nodes
-                    </div>
-                    <div class="col-md-3 dropdown dropdown-menu-right">
-                        <button class="btn btn-default dropdown-toggle" type="button" data-toggle="dropdown">
-                            Actions
-                            <span class="caret"></span>
-                        </button>
-                        <ul class="dropdown-menu" role="menu" aria-labelledby="dropdownMenu1">
-                            <li role="presentation"><a role="menuitem" tabindex="-1" href="#">View cluster</a>
-                            </li>
-                            <li role="presentation"><a role="menuitem" tabindex="-1" href="#">Configure cluster</a>
-                            </li>
-                            <li role="presentation"><a role="menuitem" tabindex="-1" href="#">Create cache</a>
-                            </li>
-                            <li role="presentation" class="divider"></li>
-                            <li role="presentation"><a role="menuitem" tabindex="-1" href="#">Shutdown cluster</a>
-                            </li>
-                        </ul>
-                    </div>
-                </div>
-            </div>
-        </div>
+        <tr ng-repeat="cluster in clusters track by $index">
+          <td><span class="label"
+                    ng-class="(cluster.isAvailable())?'label-success':'label-danger'">
+            {{cluster.isAvailable()?cluster.availability:'UNAVAILABLE'}}</span></td>
```

Comment:
We need to redefine this: availability is either AVAILABLE, DEGRADED or STOPPED. Next PR.